Fix recent changes in the case of a stack that grows up (on PARISC):

js/src/jsapi.cpp: In constructor 'JSRuntime::JSRuntime(JSUseHelperThreads)':
js/src/jsapi.cpp:895:5: error: 'nativeStackLimit' was not declared in this scope

This case didn't see the changes that the alternative got
See https://bugs.gentoo.org/show_bug.cgi?id=414297

    Jeroen Roovers <jer@gentoo.org>
--

--- a/js/src/jsapi.cpp	2013-05-11 21:19:34.000000000 +0200
+++ b/js/src/jsapi.cpp	2013-05-28 17:39:15.347027000 +0200
@@ -892,7 +892,7 @@
     PodZero(&atomState);
 
 #if JS_STACK_GROWTH_DIRECTION > 0
-    nativeStackLimit = UINTPTR_MAX;
+    mainThread.nativeStackLimit = UINTPTR_MAX;
 #endif
 }
 
